function_tablemultiplier.py

```diff
@@ -18,7 +18,7 @@ def average(a,b):
 """
 
 def table_multiplier(i,x,y):
```

Comment:
```suggestion, adding datatype recommendation is good practice.
def table_multiplier(i: int, x: int, y: int):
```

Reply:
Instead of print return data from function.

Reply:
Used meaningful name. 

Will check for return value